appveyor.yml

```diff
@@ -34,10 +34,7 @@ install:
   - conda update -q conda
   - conda config --add channels conda-forge
   - conda info -a
-  - "conda create -q -n test-environment python=%PYTHON_VERSION%"
-  - activate test-environment
-  - conda install pandas shapely=1.5 fiona pyproj rtree matplotlib descartes
-  - conda install pytest mock
+  - "conda create --quiet --name test-environment python=%PYTHON_VERSION% pandas --file requirements.txt --file requirements.test.txt"
```

Comment:
Using the `requirements.txt` is not fully the same, as eg `rtree` and `descartes` are not listed there as optional deps

Reply:
Ah, but they are listed in `requirements.test.txt`, so OK. 
Although it seems that for some reason pandas is missing from the required ones

Reply:
But they are in the `requirements.test.txt`, right?

Reply:
> Although it seems that for some reason pandas is missing from the required ones

Indeed. I thought about adding it there rather than specifying it here but I was not sure if that was intentional or not, should I fix it?

Reply:
The original reason seems to have been because of using the file to test with different versions of pandas: https://github.com/geopandas/geopandas/commit/1545e7ca55d8f5f438951fe77126f287fc09d1ce#diff-b4ef698db8ca845e5845c4618278f29a

(we don't use that currently any more in .travis.yml, although we could)